packages/core/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@lionweb/repository-core",
+  "version": "0.1.0",
+  "private": false,
+  "main": "src/server.ts",
+  "types": "dist/index.d.ts",
+  "typings": "dist/index.d.ts",
+  "type": "module",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LionWeb-io/lionweb-repository.git"
+  },
+  "bugs": {
+    "url": "https://github.com/LionWeb-io/lionweb-repository/issues"
+  },
+  "scripts": {
+    "dev": "nodemon dist/server.js",
+    "build": "rm -rf dist/ && tsc",
+    "alive": "echo I am alive",
+    "cleanup": "rm -rf ./node_modules",
+    "lint": "eslint src",
+    "manualtest": "node dist/test/ManualTests.js",
+    "test": "echo No tests in repository"
+  },
+  "devDependencies": {
+    "prettier": "3.1.0",
+    "@typescript-eslint/eslint-plugin": "6.13.2",
+    "@typescript-eslint/parser": "6.13.2",
+    "@types/chai": "4.3.11",
+    "@types/mocha": "10.0.1",
+    "eslint": "8.56.0",
+    "chai": "4.3.10",
+    "mocha": "10.2.0",
+    "prettier": "3.1.0",
+    "ts-node": "10.9.2",
+    "typescript": "5.2.2"
+  },
+  "dependencies": {
+    "@lionweb/repository-dbadmin": "0.1.0",
```

Comment:
I did not think you needed to explicitly add dependencies across sibling packages. I think for me it worked without adding them

Reply:
They might work locally in the development environment, but when you install a _released_ package from npm the packages are standalone (no siblings anymore) and then you need to refer all used packages.
I dislike that the development environment does not flag this as an error, but that's the way it often is.